src/components/vm/disks/vmDiskColumns.tsx

```diff
@@ -87,10 +88,29 @@ export const DiskSourceCell = ({
     const chunks: React.ReactNode[] = [];
     DISK_SOURCE_LIST.forEach(entry => addOptional(chunks, getDiskSourceValue(diskSource, entry.name), entry.name, entry.label));
 
-    return <DescriptionList isHorizontal>{chunks}</DescriptionList>;
+    return chunks;
 };
 
-export const DiskExtras = ({
+export const DiskSourceAbbrev = ({
+    diskSource,
+} : {
+    diskSource: DiskSource,
+}) => {
+    if (diskSource.file) {
+        return basename(diskSource.file);
+    } else if (diskSource.dev) {
+        return diskSource.dev;
+    } else if (diskSource.volume) {
+        return diskSource.volume;
+    } else if (diskSource.host.name && diskSource.host.port) {
+        return diskSource.host.name + ":" + diskSource.host.port;
+    } else if (diskSource.host.name) {
+        return diskSource.host.name;
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2324-d14228b1-20250918-114501-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)